release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -506,6 +506,11 @@ When configuring the `docker/config.json` file to allow pods to pull images from
 
 Node-related metrics and alerts have been enhanced to give you an earlier indication of when the stability of a node is compromised.
 
+[id="ocp-4-9-node-health-check-operator"]
+==== Deploy node health checks with the Node Health Check Operator (Technology Preview)
+
+You can use the Node Health Check Operator to deploy the NodeHealthCheck controller. The controller identifies unhealthy nodes and uses the Poison Pill Operator to remediate the unhealthy nodes.
```

Comment:
Here, `NodeHealthCheck` should be wrapped in backtick because it is the actual name of the controller.